pymarkdown/compatibility.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import sys
+
+if sys.version_info[0] == 3:
+    from io import StringIO
+    unicode = str
+
+if sys.version_info[0] == 2:
+    from StringIO import StringIO
+    unicode = unicode
```

Comment:
Some of these things I added to master a couple days ago after your last PR in https://github.com/mrocklin/pymarkdown/commit/f6832c3275345c3f313ce77d79e34fd293f54b7a